examples/client/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "fuels-example-client"
```

NIT: Can we keep the `fuel-streams` namespace?

```suggestion
name = "fuels-streams-client-examples"
```